src/Forms/Compare.cs

```diff
@@ -577,7 +585,7 @@ private void ShowTextDifference()
                 {
                     richTextBox1.SelectionStart = i;
                     richTextBox1.SelectionLength = 1;
-                    richTextBox1.SelectionBackColor = Color.Red;
+                    richTextBox1.SelectionColor = Color.Red;

```

Comment:
I aelectionColor isn't going to have same meaning if text if: ' ' ...  as SelectionBackColor does